src/components/CookieNotice/CookiePolicy.tsx

```diff
@@ -0,0 +1,48 @@
+import "./CookiePolicy.scss";
+import {useEffect, useState} from "react";
+import Portal from "components/Portal/Portal";
+import marked from "marked";
+import policyText from "./CookiePolicyText.md";
+
+interface CookiePolicyProps {
+  acceptFunction: () => void;
+  onClose: () => void;
+  show: boolean;
+  darkBackground: boolean;
+}
+
+const CookiePolicy = ({acceptFunction, onClose, show, darkBackground}: CookiePolicyProps) => {
+  const [policy, setPolicy] = useState({markdown: ""});
+
+  useEffect(() => {
+    fetch(policyText)
+      .then((response) => response.text())
+      .then((text) => setPolicy({markdown: marked(text)}));
+  }, []);
+
+  if (!show) {
+    return null;
+  }
+
+  const handleAccept = () => {
+    acceptFunction();
+  };
+
+  return (
+    <Portal onClose={onClose} darkBackground={darkBackground}>
```

Comment:
The portal has some margin at the sides because it's designed for looking good at our board with the coloured border
<img width="119" alt="Screenshot 2021-09-17 at 13 39 04" src="https://user-images.githubusercontent.com/36969812/133776608-7f49bcb9-6788-4e47-85c2-d98d521e632a.png">
l